course-mw/enrolment-actor/src/main/scala/org/sunbird/aggregate/CollectionSummaryAggregate.scala

```diff
@@ -70,6 +70,7 @@ class CollectionSummaryAggregate @Inject()(implicit val cacheUtil: RedisCacheUti
         val eData_type = if (StringUtils.equalsIgnoreCase(eventObj.get("edata_type").asInstanceOf[String], "enrol") || StringUtils.equalsIgnoreCase(eventObj.get("edata_type").asInstanceOf[String], "enroled")) "enrolment" else eventObj.get("edata_type").asInstanceOf[String]
         (eventObj.get("state"), eventObj.get("district")) -> Map("type" -> eData_type, "count" -> eventObj.get("userCount"))
       }).groupBy(x => x._1)
+        .filter(location => location._1._2 != null && location._1._1 != null) // Code to remove the null district and state
```

Comment:
We need to handle the case where the enrolment is blank but completion exists. Can we update the query and remove completions until we fix the completion query?

Reply:
Updated the druid query, removed the edata_type = "completion" query.